tests/test_hay11.rs

```diff
@@ -0,0 +1,64 @@
+use nml2::{
```

Comment:
Is the existence of this file enough to have it run in CI? (I don't see any changes to CI referencing this)

Reply:
cargo test picks up the #[test] thing